packages/yarnpkg-core/sources/Manifest.ts

```diff
@@ -150,6 +150,13 @@ export class Manifest {
     this.indent = getIndent(content);
   }
 
+  /**
+   * Replaces all backslashes with slashes
+   */
+  private normalizeSlashes(str: string) {
+    return str.replace(/\\/g, `/`) as PortablePath;
+  }
```

Comment:
I think this would make more sense as either:
- a free function at the bottom just like `getIndent` or `stripBOM` if it's not intended to be exposed to API consumers (I see it's currently private)
- a static method otherwise, as it doesn't use the instance in any way

Reply:
Fair point, extracted it out of the class